Gruntfile.js

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Grunt tasks to support running linter and unit tests.
+ *
+ *  @module Gruntfile
+ *
+ *  @copyright 2014, Digium, Inc.
+ *  @license Apache License, Version 2.0
+ *  @author Samuel Fortier-Galarneau <sgalarneau@digium.com>
+ */
+
+'use strict';
+
+module.exports = function(grunt) {
+
+  // threshold for failing coverage
+  var coverageThreshold = 80;
+
+  // Project configuration.
+  grunt.initConfig({
+    // Task configuration.
+    jshint: {
+      options: {
+        jshintrc: true
+      },
+      all: [
+        'Gruntfile.js',
```

Comment:
missing app.js, test/mock_dal/*.js, test/mock_dal/helpers/*.js, and test/mock_dal/repositories/*.js

Adding these exposes 15 linting errors.